asv_bench/benchmarks/__init__.py

```diff
@@ -29,3 +29,12 @@ def randn(shape, frac_nan=None, chunks=None):
         x.flat[inds] = np.nan
 
     return x
+
+
+def randint(low, high=None, size=None, frac_minus=None):
```

Comment:
Can we use a seed for all these random numbers? That should decrease the variance for these test results.

Reply:
See [line 9](https://github.com/pydata/xarray/pull/1851/files#diff-85a6ac883680893ebfac241d3ce89fe1L9).

jhamman was careful enough :)

Reply:
>  it means that results will vary depending upon whether we run the full benchmark suite or not, and depending upon the order in which benchmark tests are run.

Thanks for the details.
Understood. 
Done.